### DISCUSSION TITLE
Wondering about additional weather data

### DISCUSSION THREAD

**Discussion by jordancaraballo** [weather-gov/api#770] [Ideas]
Hi All,

Thanks for the API, pretty straight forward to download and interact with the data. I am wondering if you all have considered exposing more weather variables to the individual hourly forecasts? (e.g. additional variables, indices, etc.). To my knowledge these models do generate an additional set of variables that I have not seen when looking at the individual data from the API.

Thanks,
Jordan

**Comment by sullynole** [comment]
The forecastGridData endpoint contains all of the raw data sent up by each WFO.  Some offices may send more elements than others.  While models often are used to initialize data, the gridpoints endpoints data are not model output.  Each of the elements are analyzed and adjusted by forecasters with software tools at the local weather forecast offices. These data are considered the NWS forecast.
